roles/confluent.control_center/templates/control-center.properties.j2

```diff
@@ -1,5 +1,5 @@
 # Control Center Configuration
-confluent.controlcenter.rest.listeners={{control_center_http_protocol}}://0.0.0.0:{{control_center_port}}
+confluent.controlcenter.rest.listeners={{control_center_http_protocol}}://{{control_center_listener_hostname}}:{{control_center_port}}
```

Comment:
@JumaX do you like this variable name?
could also be "control_center_hostname"

Reply:
Looks good, its nice you can split out the name to be unique, especially on a host with multiple DNS entries.  This is good!